Sources/MockoloFramework/Parsers/SwiftSyntaxExtensions.swift

```diff
@@ -368,11 +368,48 @@ extension VariableDeclSyntax {
                 typeName = vtype
             }
 
+            let storageType: VariableModel.MockStorageType
+            switch v.accessorBlock?.accessors {
```

Comment:
This block is very complicated 🤪 but mockolo handles both class and protocol here, so it has to be this way.